src/main.js

```diff
@@ -0,0 +1,242 @@
+function openNav() {
+  document.getElementById('myNav').style.display = 'block';
+}
+
+function closeNav() {
+  document.getElementById('myNav').style.display = 'none';
+}
+
+document.querySelector('.toggle').addEventListener('click', openNav);
+document.querySelector('.closebtn').addEventListener('click', closeNav);
+document.querySelectorAll('.pop-nav').forEach((link) => {
+  link.addEventListener('click', () => {
+    document.getElementById('myNav').style.display = 'none';
+  });
+});
+
+/* const projectname = document.querySelector('.projectname'); */
```

Comment:
- [ ] [OPTIONAL] Leaving unused codes as inline comments in your code makes the base messy and somewhat difficult to read. Kindly remove this comment to make your work look professional and clean. :muscle: